spec/controllers/restaurant_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  let(:restaurant_hash) do
+    {
+      name: 'Name',
+      genre: :sushi,
+      rating: 1,
+      accepts_10bis: true,
+      delivery_time: :slow
+    }
+  end
+  let(:keys) { restaurant_hash.keys.to_a }
+  before (:each) do
+    Restaurant.create(restaurant_hash)
+  end
+
+  describe 'validations' do
```

Comment:
'validations' is wrong in any case (copy+pasted from the model spec...). 
I've renamed it in a future commit to 'actions', but separating makes a lot of sense.